td.vue/src/components/Navbar.vue

```diff
@@ -14,6 +14,12 @@
       </b-navbar-nav>
       
       <b-navbar-nav class="ml-auto">
+        <b-nav-item>
+          <b-button size="sm" variant="outline-light" @click="handleToggleDarkMode">Toggle Dark Mode</b-button>
+        </b-nav-item>
+        <b-nav-item>
+          <b-button size="sm" variant="outline-light" @click="handleSetSystemTheme">System Theme</b-button>
```

Comment:
is this 'set system theme' needed? there is the dark mode toggle, which will provide the system theme when off